server/readme.md

```diff
@@ -1,3 +1,42 @@
-# Book Api
+# Flask (sqlite) Book Server
 
-> uh oh.. I have bugs
\ No newline at end of file
+Starting off, run the following commands for the initial setup:
+```js
+pipenv shell
+pipenv install
+```
+
+To start the server up run the following inside the pipenv shell:
+```js
+python app.py
+```
+
+New submissions must following format:
+```json
+{
+    "title": "string",
+    "author": "string",
+    "url": "string",
+    "genre": "string",
+    "star_rating": "string",
+    "book_read": boolean
+}
+```
+
+The following addresses will give access to the CRUD:
+- POST
+```js
+http://localhost:5000/add-book
+```
+- PATCH
+```js
+http://localhost:5000/book-read/<id>
+```
+- GET
+```js
+http://localhost:5000/books
+```
+- DELETE
+```js
+http://localhost:5000/delete-book/<id>
+```
```

Comment:
# Flask (sqlite) Book Server


Starting off, run the following commands in your terminal while in the `server` folder for the initial setup:
```
$ pipenv install
$ pipenv shell
```

Run the following in your terminal while inside the `server` folder to start your server:
```
$ pipenv shell
$ python app.py
```

New submissions must have the following format:
```json
{
    "title": "string",
    "author": "string",
    "url": "string",
    "genre": "string",
    "star_rating": "string",
    "book_read": boolean
}
```

The following addresses will give access to the CRUD:
- POST
```
http://localhost:5000/add-book
```

- PATCH
```
http://localhost:5000/book-read/<id>
```

- GET
```
http://localhost:5000/books
```

- DELETE
```
http://localhost:5000/delete-book/<id>
```

Reply:
Please get rid of the `js` portion of your three back-tick code blocks and put single back-tick around the word `server`.

Please add the `$` in front of each command that will be run in the terminal.

We might want to add some extra information on each of the CRUD routes.  We might want to let the user know if a `body` is required.  For instance, the `POST` needs the entire `schema` and the `GET` doesn't require anything.